### ISSUE TITLE
opbeat-js version 2.0.1 on npm registry?

### ISSUE THREAD

**Issue by juliusosokinas** [opbeat/opbeat-js#49]
Where this version is coming from? I don't see anything on repository, but version is reflected here: https://www.npmjs.com/package/opbeat-js.

Can confirm this was installed:

```
opbeat-js@2.0.1 node_modules/opbeat-js
├── simple-lru-cache@0.0.2
├── es6-promise@3.1.2
├── loglevel@1.4.0
├── error-stack-parser@1.3.5 (stackframe@0.3.1)
├── stack-generator@1.0.7 (stackframe@0.3.1)
├── event-stream@3.3.2 (stream-combiner@0.0.4, duplexer@0.1.1, from@0.1.3, pause-stream@0.0.11, map-stream@0.1.0, split@0.3.3, through@2.3.8)
└── zone.js@0.5.15
```

**Comment by juliusosokinas** [comment]
OK, I see everything now. I'm pretty sure branch and versions weren't there a moment ago... Closing.

**Comment by jahtalab** [comment]
Thanks for reaching out.
I just published [v2.0](https://github.com/opbeat/opbeat-js/tree/v2.0) branch.
